src/helper/bitmap.js

```diff
@@ -1,5 +1,85 @@
 import paper from '@scratch/paper';
 
+const line = function (point1, point2, callback) {
```

Comment:
Might want to rename this to something like `forEachLinePoint`, since this doesn't actually draw a line.

Reply:
Done